### PR TITLE
order code

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

**Comment by AdiZav** [comment]
I guess the workflow needs to be updated to check only pr.
At least it works :)

File: main.py [hunk 1]
```diff
@@ -3,93 +3,35 @@
...
-    commit = repo.get_commit(os.environ.get("GITHUB_SHA"))
+    commit = repo.get_commit(os.environ["GITHUB_SHA"])
     try:
-        issue_number = data["number"]
+        pull = repo.get_pull(data["number"])
     except KeyError:
-        issue_number = data["issue"]["number"]
+        pull = repo.get_pull(data["issue"]["number"])
+
+    if event_type in ("pull_request_target", "pull_request"):
```
Comment (myakove) [inline main.py:27]: we need, this need to be done on pull_request and _target, with your suggestion we will need to call it multiple times

Comment (AdiZav) [reply main.py]: That's why you create different workflows that trigger on different events

**Review by AdiZav — APPROVED** [review]

**Pull request merged**